src/main/java/com/revature/services/EvalLogicImpl.java

```diff
@@ -16,8 +22,93 @@
 	@Autowired
 	private EvalRepository dao;
 	
+	@Autowired
+	private EvalCommentRepository commentDao;
+	
+	@PersistenceContext
+    private EntityManager entityManager;
+	
 
 //CREATE----------------------------
+	@Override
+	@Transactional
+	public Eval createEval(Eval eval) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "createEval" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)